modules/gdscript/gdscript.cpp

```diff
@@ -912,6 +912,11 @@ void GDScript::unload_static() const {
 }
 
 Variant GDScript::callp(const StringName &p_method, const Variant **p_args, int p_argcount, Callable::CallError &r_error) {
+	if (p_method == SNAME("new")) {
+		// Constructor.
+		return _new(p_args, p_argcount, r_error);
+	}
```

Comment:
There is a bound method, it doesn't work?

https://github.com/godotengine/godot/blob/bf5582660b9d4c47a14854b22981f671b5aa29d7/modules/gdscript/gdscript.cpp#L1078-L1080

Reply:
maybe i was a bit pedantic, but i copied it from `GDScriptNativeClass::callp()` and kept it before everything else so its behavior cant be changed by a static native method called 'new'

Reply:
i think i found a much better way, by moving the native call after `Script::callp()`